<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сохранение</string>
  </property>
  <widget class="QPushButton" name="btn_chek">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_all">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Всех существах</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_one">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Одном существе</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Вы хотите сохранить информацию о:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите формат, в котором вы хотите сохранить файл:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_exe">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>xlsx</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_csv">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>csv</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Вы можете сохранить информацию об одном существе 
или о всех имеющихся в формате exe и csv. 
Информация о картинке не будет сохранена в файл.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
